### ISSUE TITLE
Abteilung

### ISSUE THREAD

**Issue by JeVaske** [SVWS-NRW/Schild3-BetaTest#793]
v3.0.75 Für die Zugriffsrechte in SchildWeb war in dem Katalog Klassengruppen/Abteilungen der Eintrag beider Koordinatoren der Erprobungs-und Mittelstufe notwendig. Folglich werden im neuen Filter Abteilung, den ich sehr proktisch finde, Erprobungssufe und Mittelstufe doppelt aufgelistet.
![Foto7](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/78501680/8ad7ba1d-b139-4327-a60b-b4d450f63eb7)

**Comment by AnneSchueller** [comment]
d.h. ihr habt 2 mal den Eintrag "Erprobungsstufe" unter Abteilungen, die sich jeweils nur bezüglich der Lehrkraft unterscheiden?
Um das geschickt zu lösen, müsste man mehrere Koordinatoren einer Ableitung zuordnen können. Dies ist m.E. aktuell nicht möglich mit der Tabellenstruktur. 

![image](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/79116896/c20886ce-4dad-46a7-8e05-7118ecf0b588)

Wenn man dann mehrere Einträge erzeugt (um mehrere Koordinatoren speichern zu können), finde ich es sinnvoll, wenn hier auch beide Einträge als Filterung zur Auswahl stehen. Der Anwender könnte ja die Bezeichnung "Erprobungsstufe" um das Lehrerkürzel erweitern. 
Würde man bei zwei gleichen Einträgen für "Erprobungsstufe" nur eine Auswahlmöglichkeit für den Filter haben, müsste man klären, was passiert, wenn trotz gleicher Bezeichnung unterschiedliche Klassen angegeben sind.

Ich würde das Issue gerne schließen.

**Comment by JeVaske** [comment]
Ja, ganz genau. Es sind zwei separate Einträge mit dem Eintrag "Erprobungsstufe" eingetragen. Mir persönlich ist das auch nicht mehr wichtig, weil wir leider Schildweb seit neuestem nicht mehr nutzen. Ich denke aber, dass dieser Filter bei vielen Ribekakunden etwas seltsam aussehen wird.

**Comment by AnneSchueller** [comment]
Hier müssten die Anwender in Schild die Bezeichnungen anpassen wenn es mehrere Koordinatoren mit dem gleichen Eintrag gibt.
Ich schließe das Issue mal.
